source/updateCheck.py

```diff
@@ -413,7 +413,7 @@ def _error(self):
 			tip = pgettext(
 				"updateCheck",
 				# Translators: A suggestion of what to do when fetching add-on data from the store fails and the default metadata URL is being used.
```

Comment:
There is now no suggestion of what to do.
```suggestion
				# Translators: Presented when fetching add-on data from the store fails and the default metadata URL is being used.
```